src/main.ts

```diff
@@ -33,7 +33,14 @@ const WEBHOOK_SECRET: string | undefined = Deno.env.get("WEBHOOK_SECRET");
 await Deno.mkdir(`${config.workDir}/repo`, { recursive: true });
 await Deno.mkdir(`${config.workDir}/tmprdf`, { recursive: true });
 await Deno.mkdir(`${config.workDir}/tmpttl`, { recursive: true });
-await createBadge("Unknown");
+
+const db = new JobsDataBase(`${config.workDir}/jobs`);
```

Comment:
Why not take care of the badge within the worker?

Reply:
badge is made by the worker, this is only on initial startup